### PR TITLE
dev-to-alpha

### PR DESCRIPTION
**Pull request by szuecs**
* **Fix Skipper: websocket bug in load balanced backends**
* **Fix/redis based csluterratelimits**
   <sup>Merge pull request #1900 from zalando-incubator/fix/redis-based-csluterratelimits</sup>
* **Update skipper with redis based cluster rate limiting**
   <sup>Merge pull request #1898 from zalando-incubator/update-skipper</sup>
* **Disabled acces log by default**
   <sup>Merge pull request #1886 from zalando-incubator/disable-skipper-access-logs</sup>
* **Update CLC**
   <sup>Merge pull request #1896 from zalando-incubator/update-clc</sup>
* **Ndots kube system**
   <sup>Merge pull request #1875 from zalando-incubator/ndots-kube-system</sup>
* **Signed-off-by: Alexey Ermakov <alexey.ermakov@zalando.de>
  **
   <sup>Merge branch 'dev' into ndots-kube-system</sup>
* **Signed-off-by: njuettner <nick@zalando.de>
  **
   <sup>Merge branches 'dev' and 'ndots-kube-system' of github.com:zalando-incubator/kubernetes-on-aws into ndots-kube-system</sup>

### CODE REVIEW

**Comment by aermakov-zalando** [comment]
Does it include the websocket fix as well?

**Comment by angelaigreja** [comment]
It does because the Skipper version is after the merge.

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -16,9 +16,6 @@ skipper_limits_mem: "250Mi"
 skipper_requests_cpu: "150m"
 skipper_requests_mem: "50Mi"
 
-# skipper general settings
-skipper_disable_access_logs: "false"
```
Comment (angelaigreja) [inline cluster/config-defaults.yaml]: Is this intended?

**Comment by szuecs** [comment]
@aermakov-zalando Yes, sorry I only tracked it in the internal issue tracker

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**